lib/Components/Search/Search.js

```diff
@@ -4,15 +4,38 @@ import './Search.css'
 export default class Search extends React.Component {
   constructor(props) {
     super(props);
-    this.state = ({
+    this.state = {
       input: '',
-    });
+    };
   }
 
+  handleChange(e) {
+    this.setState({
+      input: e.target.value,
+    })
+  }
+
+  getLocation(userInput) {
+    let splitInput = userInput.split(', ');
```

Comment:
@christielynam I don't think we need this anymore, thoughts?? The api is super loose and let's us do whatever the F we want.